### DISCUSSION TITLE
Taro PlayGround

### DISCUSSION THREAD

**Discussion by RafaelCENG** [wuba/react-native-echarts#63] [General]
Is Taro only in chinese?

**Comment by zhiqingchen** [comment]
The taro project mainly for Chinese users, supports WeChat Mini Programs, web and APP (use React Native) development. Global users are recommended to use expo, which supports React Native and web. taro Playground currently only does Chinese support. If you wish to run the react native ECharts demo, refer to https://wuba.github.io/react-native-echarts/docs/getting-started/run-example

If you just look at the demo of the chart, download the app and on this page you can see Charts.
![image](https://github.com/wuba/react-native-echarts/assets/1876158/f14bbece-b79e-49a8-a419-f6e2fbc48cf9)

**Reply by RafaelCENG** [reply]
Thanks!!
